encode.js

```diff
@@ -1227,5 +1234,8 @@ export const { NEVER, ALWAYS, DECIMAL_ROUND, DECIMAL_FIT } = FLOAT32_OPTIONS
 export const REUSE_BUFFER_MODE = 512
 export const RESET_BUFFER_MODE = 1024
 export const THROW_ON_ITERABLE = 2048
-
-
+const lexSortFn = (a, b) => {
+	if (a.length < b.length) return -1
```

Comment:
Does the CBOR spec say that length takes precedence over bytewise/lexigraphic order? I don't think it does, does it?

Reply:
It actually does: https://www.rfc-editor.org/rfc/rfc8949.html#name-length-first-map-key-orderi